src/test/java/uk/gov/pay/directdebit/webhook/gocardless/services/handlers/GoCardlessEventHandlerTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.pay.directdebit.webhook.gocardless.services.handlers;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.pay.directdebit.events.model.GoCardlessEvent;
+import uk.gov.pay.directdebit.mandate.services.MandateQueryService;
+import uk.gov.pay.directdebit.payments.services.PaymentQueryService;
+
+import java.util.List;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static uk.gov.pay.directdebit.events.model.GoCardlessResourceType.MANDATES;
+import static uk.gov.pay.directdebit.events.model.GoCardlessResourceType.PAYMENTS;
+import static uk.gov.pay.directdebit.events.model.GoCardlessResourceType.REFUNDS;
+import static uk.gov.pay.directdebit.events.model.GoCardlessResourceType.SUBSCRIPTIONS;
+import static uk.gov.pay.directdebit.payments.fixtures.GoCardlessEventFixture.aGoCardlessEventFixture;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GoCardlessEventHandlerTest {
+
+    @Mock
+    private MandateQueryService mockedMandateQueryService;
+
+    @Mock
+    private PaymentQueryService mockPaymentQueryService;
+
+    @InjectMocks
+    private GoCardlessEventHandler goCardlessMandateHandler;
+
+    @Test
+    public void shouldNotHandleEventsWithRefundsResource() {
+        GoCardlessEvent goCardlessEvent = aGoCardlessEventFixture().withResourceType(REFUNDS).withAction("created").toEntity();
+        goCardlessMandateHandler.handle(List.of(goCardlessEvent));
+        verify(mockedMandateQueryService, never()).findByGoCardlessMandateIdAndOrganisationId(any(), any());
+        verify(mockPaymentQueryService, never()).findByGoCardlessPaymentIdAndOrganisationId(any(), any());
```

Comment:
I think the important part here is that there’s no interactions with the `UserNotificationService`.

Reply:
fixed